x/incentives/keeper/gauge_test.go

```diff
@@ -44,6 +47,10 @@ var (
 		sdk.NewCoin("uosmo", osmomath.NewInt(100000)),
 		sdk.NewCoin("atom", osmomath.NewInt(99999)),
 	)
+
+	customGroupCreationFee = sdk.NewCoins(sdk.NewInt64Coin(feeDenom, 1000000))
+
+	errorNoCustomFeeInBalance = fmt.Errorf("0%s is smaller than %s: insufficient funds", feeDenom, customGroupCreationFee)
```

Comment:
Out of curiosity, why don't we have a custom error type here? Is it just to avoid PR scope creep with driveby changes?

Reply:
This is because the error originates in the bank module which does not export it.

As a result, I'm forced to redefine here and do error string comparison as opposed to proper error comparison